yellowbrick/features/jointplot.py

```diff
@@ -145,7 +147,6 @@ def __init__(self, ax=None, feature=None, target=None,
         self.joint_args = joint_args
         self.xy_plot = xy_plot
         self.xy_args = xy_args
-        self.size = (size, size)
```

Comment:
Setting size like this, although it kept the image square also interfered with the plot directive. So now the joint plot is 9x6 ... which is weird looking. I'll need to figure something out to fix it.

Reply:
I think that was similar to what I about different dimensions over in [446](https://github.com/DistrictDataLabs/yellowbrick/pull/446#discussion_r252865766)

Reply:
Maybe something like what we did w/ the residuals plot when we introduced the histogram? See also #214 and #605

Reply:
Yes, I think that's exactly the solution! I can get started on that but it's going to require a fairly significant overhaul of the JointPlot code, so I didn't want to do it in this PR (and that's what I was being blocked on by forgetting to create a feature branch ...).